conf/machine/include/stm32mp1-common.inc

```diff
@@ -3,6 +3,11 @@ DEFAULTTUNE = "cortexa7thf-neon-vfpv4"
 PREFERRED_PROVIDER_virtual/bootloader ??= "u-boot-mainline"
 PREFERRED_PROVIDER_virtual/kernel ??= "linux-mainline"
 
+# With kernels >=5.2  systemd-networkd is unable to bring up the link.
+# So backport upstream systemd in yocto master
```

Comment:
So backport upstream systemd recipe (from Yocto master) in our warrior branch.

Reply:
Apart  that:

Tested-by: Pierre-Jean Texier <pjtexier@koncepto.io>

Reply:
Fixed ;)